onnx/backend/test/cpp/driver/test_driver.h

```diff
@@ -0,0 +1,79 @@
+#include <iostream>
+#include <queue>
+#include <string>
+#include <cstdlib>
+#include <cstdio>
+
+#include "onnx/onnx_pb.h"
+#include "onnx/proto_utils.h"
+#include "onnx/onnxifi_loader.h"
+#include "onnxifi.h"
+
+#include "gtest/gtest.h"
+
+struct TestData{
+	std::vector<std::string> input_filenames_;
+	std::vector<std::string> output_filenames_;
+
+	TestData(){}
+
+	TestData(const std::vector<std::string> &input_filenames, const std::vector<std::string> &output_filenames){
+		input_filenames_ = input_filenames;
+		output_filenames_ = output_filenames;
+	}
+};
+struct TestCase{
+	TestCase(
+		const std::string& model_filename,
+		const std::string& model_dirname,
+		const std::vector<TestData>& test_data){
```

Comment:
You can use the ctor initializer to initialize the members. 

https://en.cppreference.com/w/cpp/language/initializer_list